Cargo.toml

```diff
@@ -36,11 +36,11 @@ generic-simd = ["bytecount/generic-simd"]
 annotate-snippets = { version = "0.9", features = ["color"] }
 anyhow = "1.0"
 bytecount = "0.6"
-cargo_metadata = "0.14"
+cargo_metadata = "0.15.4"
 clap = { version = "4.2.1", features = ["derive"] }
 diff = "0.1"
-dirs = "4.0"
-env_logger = "0.9"
+dirs = "5.0.1"
```

Comment:
I'm not sure about this one. We've seen cases where major version updates of this package result in (understandably) breaking changes around the directories that are searched, which in turn can break/impact rustfmt's config file behavior.

I'd rather drop that from this one and review/investigate/discuss a `dirs` update separately